kubernetes/resource_kubernetes_stateful_set_test.go

```diff
@@ -575,7 +614,7 @@ func testAccKubernetesStatefulSetConfigUpdateImage(name string) string {
           }
 
           volume_mount {
-            name       = "workdir"
+            name       = "ss-test"
```

Comment:
All these replacement changes are because this config was invalid and causing failures where `wait_for_rollout` was set to true.